### PR TITLE
Incorporate no_docs_url option and update CI tests

### PR DESCRIPTION
**Pull request by anzoman**
This PR adds `--no-docs-url` and updates CI integration tests accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ The action accepts the following inputs:
 | `upload_values`   | no       | false   | Parses and uploads values from Ansible task parameters to the backend.                                                                                                             |                                
 | `upload_metadata` | no       | false   | Uploads metadata (i.e., file names, line and column numbers) to the backend.                                                                                                       |                                
 | `display_level`   | no       | hint    | Displays check results with specified level or greater (e.g., warning will show all warnings and errors, but suppress hints). Available options: hint, warning, error.             |
+| `no_docs_url`     | no       | false   | Disables outputting URL to documentation.                                                                                                                                          |
```
Comment (matejart) [inline README.md:46]:
```suggestion
| `no_docs_url`     | no       | false   | Omits documentation URLs from the output.                                                                                                                                          |  
```

File: action.yml [hunk 2]
```diff
@@ -24,6 +24,10 @@ inputs:
     description: "Displays check results with specified level or greater (e.g., warning will show all warnings and errors, but suppress hints). Available options: hint, warning, error."
     required: false
     default: "hint"
+  no_docs_url:
+    description: "Disables outputting URL to documentation."
```
Comment (matejart) [inline action.yml:28]:
```suggestion
    description: "Omits documentation URLs from the output."

```

**Review by matejart — CHANGES_REQUESTED** [review]
Thank you, @anzoman! Looks good except for a bit of wording changes to be consistent with the one in the CLI's documentation.

File: README.md (hunk 1 above)
Comment (anzoman) [reply README.md]: Done.

File: action.yml (hunk 2 above)
Comment (anzoman) [reply action.yml]: Done.

**Review by matejart — APPROVED** [review]

**Pull request merged**